docs/03-Metrics/hubble_metrics.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 1
+---
+
+# Hubble Metrics
+
+When Retina is deployed with Hubble control plane, the metrics include Node-level and Pod-level. Metrics are stored in Prometheus format, and can be viewed in Grafana.
+
+## Metrics
+
+* Node-Level Metrics: These metrics provide insights into traffic volume, dropped packets, number of connections, etc. by node.
+* Hubble Metrics (DNS and Pod-Level Metrics): These metrics include source and destination pod information allowing to pinpoint network-related issues at a granular level. Metrics cover traffic volume, dropped packets, TCP resets, L4/L7 packet flows, etc. DNS metrics include DNS errors and DNS requests missing responses.
+
+### Node-Level Metrics
+
+The following metrics are aggregated per node. All metrics include labels:
+
+* `cluster`
+* `instance` (Node name)
+
+Retina provides metrics for both Linux and Windows operating systems.
+The table below outlines the different metrics generated.
+
+| Metric Name                                    | Description | Extra Labels | Linux | Windows |
+|------------------------------------------------|-------------|--------------|-------|---------|
+| **networkobservability_forward_count**         | Total forwarded packet count | `direction` | ✅ | ✅ |
+| **networkobservability_forward_bytes**         | Total forwarded byte count | `direction` | ✅ | ✅ |
+| **networkobservability_drop_count**            | Total dropped packet count | `direction`, `reason` | ✅ | ✅ |
+| **networkobservability_drop_bytes**            | Total dropped byte count | `direction`, `reason` | ✅ | ✅ |
+| **networkobservability_tcp_state**             | TCP currently active socket count by TCP state. | `state` | ✅ | ✅ |
+| **networkobservability_tcp_connection_remote** | TCP currently active socket count by remote IP/port. | `address` (IP), `port` | ✅ | ❌ |
+| **networkobservability_tcp_connection_stats**  | TCP connection statistics. (ex: Delayed ACKs, TCPKeepAlive, TCPSackFailures) | `statistic` | ✅ | ✅ |
+| **networkobservability_tcp_flag_counters**     | TCP packets count by flag. | `flag` | ❌ | ✅ |
+| **networkobservability_ip_connection_stats**   | IP connection statistics. | `statistic` | ✅ | ❌ |
+| **networkobservability_udp_connection_stats**  | UDP connection statistics. | `statistic` | ✅ | ❌ |
+| **networkobservability_udp_active_sockets**    | UDP currently active socket count |  | ✅ | ❌ |
+| **networkobservability_interface_stats**       | Interface statistics. | InterfaceName, `statistic` | ✅ | ✅ |
+
+### Pod-Level Metrics (Hubble Metrics)
+
+The following metrics are aggregated per pod (node information is preserved). All metrics include labels:
+
+* `cluster`
+* `instance` (Node name)
+* `source` or `destination`
```

Comment:
Technically it is possible for a given metric to have both source and destination labels populated, it is just that we configured the default charts that way (sourceEgress and destinationIngress). We should clarify it IMO.